web/client/components/data/identify/__tests__/DefaultViewer-test.jsx

```diff
@@ -268,4 +268,16 @@ describe('DefaultViewer', () => {
         expect(gfiViewer.childNodes[1]).toEqual(swipeableView);
 
     });
+
+    it('creates the DefaultViewer component with no map features and returns an alert', () => {
```

Comment:
will do

Reply:
I'm assuming the reason you suggested I place my test right after this is because the `buildIdentifyRequest` function has a variable `featureInfo` which would adjust depending on whether the vector has feature information or not. 

After testing it out, the `featureInfo` variable is always an empty object irrespective of whether the vector file has additional information or not. The object returned is also always the same, irrespective of whether the vector file has feature information or not. I used [shape file](https://www.naturalearthdata.com/http//www.naturalearthdata.com/download/110m/physical/ne_110m_lakes.zip) (the lakes data you shared with me) as my basetest case. I will keep looking

Reply:
I didn't understand exactly what you mean. 
The problem was that a geoJSON without properties (like the one in the test map) is not correctly handled to retrieve the properties, because it fails in the point I fixed.
I think you have simply to create a unit test that the `buildIdentifyRequest` doesn't fail if properties are not defined for the fist feature of the geoJSON (as your fix does).